js/gateio.js

```diff
@@ -408,7 +408,8 @@ module.exports = class gateio extends Exchange {
                     'spot': 'spot',
                     'margin': 'margin',
                     'cross_margin': 'cross_margin',
-                    'crossMargin': 'cross_margin',
```

Comment:
The idea behind this was consistency, because everything else we have is camelcase

Reply:
Not for exchange-specific account names, though. It's very hard to guess `'crossMargin'` without looking into the code. All others could be guesses by looking at either CCXT docs or the exchange API docs.